include/can/core/freertos_sender_task.hpp

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include <array>
+
+#include "FreeRTOS.h"
+#include "can/core/can_bus.hpp"
+#include "can/core/message_core.hpp"
+#include "can/core/messages.hpp"
+#include "common/core/logging.hpp"
+#include "common/core/message_queue.hpp"
+
+namespace freertos_sender_task {
+
+struct TaskMessage {
+    uint32_t arbitration_id;
+    can_messages::ResponseMessageType message;
+};
+
+/**
+ * Entry point for a CAN sender class.
+ * @tparam QueueImpl
+ */
+template <template <class> class QueueImpl>
+requires MessageQueue<QueueImpl<TaskMessage>, TaskMessage>
+class MessageSenderTask {
```

Comment:
so any task that needs to send messages back to the can bus would build a message sender task?

Reply:
No. There'll be one in each subsystem. It will be shared among all tasks.

Reply:
I see

Reply:
In that case we probably want to delete the copy constructor (might have to change the builder functions to return with `std::move`)

Reply:
to be clear, by "deleting the copy constructor" I mean explicitly doing so by doing `MessageSenderTask(const MessageSenderTask&) = delete;` and `(...) operator=(...) = delete;`